docs/reference/client-worker-configuration.md

```diff
@@ -50,4 +49,50 @@ The file may be called by any name but should have the following structure:
+| `loginRateLimit` | Optional | Defines an upper limit for the frequency at which worker instances of this worker type are allowed to connect to a single deployment. It contains two fields: <br> - `duration`: time window of the login rate limit to be specified as a string in the form of `<x>h<y>m<z>s`, e.g. `1h30s` <br> - `requestsPerDuration`: Maximum number of worker instances that is allowed to connect during the given duration <br><br> For more information, see [Worker connection limits](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-rate-limit-capacity). |
+| `connectionCapacityLimit` | Optional | Defines an upper limit for how many worker instances of this worker type may be simultaneously connected to a single deplyoment. It has a single field (`maxCapacity`) for specifying the maximum number of connected worker instances. <br><br> For more information, see [Worker connection limits](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-rate-limit-capacity). |
+
+#### Upload Configuration
+
+The upload configuration specifies how to start a client-worker with the [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher). Below is an example excerpt:
+```json
+      "windows": {
+        "localBuildDirectory": "bin/x64/ReleaseWindows",
+        "launcherCommand": "MyWorker.exe",
+        "launcherArguments": [
+          "${IMPROBABLE_RECEPTIONIST_HOST}",
+          "${IMPROBABLE_RECEPTIONIST_PORT}",
+          "${IMPROBABLE_WORKER_ID}",
+          "--my_additional_flag=true"
+        ]
+      },
+      "macos": {
+        "localBuildDirectory": "bin/x64/ReleaseMacOS",
+        "launcherCommand": "MyWorker.app",
+        "launcherArguments": [
+          "${IMPROBABLE_RECEPTIONIST_HOST}",
+          "${IMPROBABLE_RECEPTIONIST_PORT}",
+          "${IMPROBABLE_WORKER_ID}",
+          "--my_additional_flag=true"
+        ]
+      },
+      "linux": {
+        "localBuildDirectory": "bin/x64/ReleaseLinux",
+        "launcherCommand": "MyWorker",
+        "launcherArguments": [
+          "${IMPROBABLE_RECEPTIONIST_HOST}",
+          "${IMPROBABLE_RECEPTIONIST_PORT}",
+          "${IMPROBABLE_WORKER_ID}",
+          "--my_additional_flag=true"
+        ]
+      }
+```
+
+There must be a configuration for each platform the worker will be run on. The platform name can be one of `windows`, `macos` or `linux`. Each of these can have the following fields:
+
+| Field | Required/Optional | Description | 
+| :------------- | :------------- | :------- |
+| `localBuildDirectory` | Required | The directory containing the binary to be executed. <br><br> **Note:** Your built-out worker must be self-contained in the `localBuildDirectory` and all files that your worker might depend on while running must be in this folder. Files outside this folder will not be uploaded to the cloud as part of your project assembly. |
```

Comment:
```suggestion
| `localBuildDirectory` | Required | The directory containing the binary to be executed. <br><br> **Note:** Your built-out worker instance must be self-contained in the `localBuildDirectory` and all files that your worker instance might depend on while running must be in this folder. Files outside this folder will not be uploaded to the cloud as part of your project assembly. |
```
Missed these last time around - I think "instance" is correct here?

Reply:
"Worker" in this context is referring to the build artefacts/files that comprise the worker. Happy to discuss in person.